Sources/CocoaLumberjack/DDLog.m

```diff
@@ -1015,6 +1015,10 @@ - (void)dealloc {
 
 @implementation DDLogMessage
 
+// Fix @selectors name and type conflict from AppKit/UIKit (e.g. views NSInteger tag property)
+// without refactoring DDLog interal code.
+@synthesize representedObject = _tag;
```

Comment:
If we do this, I'd rather do it "correctly" and rename it throughout our codebase (Xcode's refactoring tools should be able to do this). Otherwise we'll always have the confusion of "why is this named tag internally?" or "where's the `_representedObject` backing the `representedObject`?"...